src/controllers/ChooseAppealReasonController.ts

```diff
@@ -26,10 +30,23 @@ const navigation: Navigation = {
     }
 };
 
-@controller(CHOOSE_REASON_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON), SessionMiddleware, AuthMiddleware)
-export class ChooseAppealReasonController extends BaseController<any>{
+interface FormBody {
+    reason: ReasonType;
+}
+
+@controller(CHOOSE_REASON_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON), SessionMiddleware, AuthMiddleware,
+CompanyAuthMiddleware)
```

Comment:
can you please update the `AppliedCompanyAuthMiddleware.test.ts` with this page

Reply:
Ah yes, good spot.